packages/node-cli/src/validation/InteractiveAnswersValidator.js

```diff
@@ -18,10 +18,10 @@ const { ANSWERS_VALIDATION_MESSAGES, COMMAND_OPTION_IS_MANDATORY } = require('..
 const ALPHANUMERIC_LOWERCASE_REGEX = '[a-z0-9]+';
 const ALPHANUMERIC_LOWERCASE_WHOLE_REGEX = `^${ALPHANUMERIC_LOWERCASE_REGEX}$`;
 const ALPHANUMERIC_HYPHEN_UNDERSCORE = /^[a-zA-Z0-9-_]+$/;
+const ALPHANUMERIC_HYPHEN_UNDERSCORE_EXTENDED = /^[a-zA-Z0-9-_ ]+[a-zA-Z0-9-_ .]*$/;
```

Comment:
I see that this has been a decision of the story but still there are plenty of characters that are allowed in FileCabinet and in the OS that are allowed.

I think we should stick with the values allowed in the FileCabinet since there could be clients that use these characters (not usual but still there could be some exceptions)